src/Simulation/TargetDefinitions/Interfaces/IIntrinsicIsingXX.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.Quantum.Intrinsic.Interfaces
 {
-    public interface IIntrinsicIsingXX : IOperationFactory
+    public interface IIntrinsicApplyUncontrolledRxx : IOperationFactory
```

Comment:
FYI: The identifier is renamed but the file is not.

Reply:
When renaming, prefer `git mv` (to preserve the history) rather than other commands or renaming in IDE.

Reply:
Good catch!